packages/now-routing-utils/src/superstatic.ts

```diff
@@ -123,7 +123,7 @@ export function sourceToRegex(
   source: string
 ): { src: string; segments: string[] } {
   const keys: Key[] = [];
-  const r = pathToRegexp(source, keys, { strict: true });
+  const r = pathToRegexp(source, keys, { strict: true, sensitive: true });
```

Comment:
```suggestion
  const r = pathToRegexp(source, keys, { strict: true, sensitive: true, delimiter: '/' });
```
We should probably match the delimiter with what we use in Next.js

Reply:
Looks good! Seems the default was `/#?`.

The strange part is that it didn't break any of the tests like I would expect 🤔

Reply:
Ok tests updated 👍